github/apps.go

```diff
@@ -32,8 +36,21 @@ type App struct {
 
 // InstallationToken represents an installation token.
 type InstallationToken struct {
-	Token     *string    `json:"token,omitempty"`
-	ExpiresAt *time.Time `json:"expires_at,omitempty"`
+	Token        *string                  `json:"token,omitempty"`
+	ExpiresAt    *time.Time               `json:"expires_at,omitempty"`
+	Permissions  *InstallationPermissions `json:"permissions,omitempty"`
+	Repositories *[]Repository            `json:"repositories,emitempty"`
```

Comment:
Please change `*[]Repository` to `[]*Repository` instead.

Reply:
Done.